packages/react-iv-viewer/src/types.ts

```diff
@@ -0,0 +1,46 @@
+import { ImageViewer as BaseImageViewer } from 'iv-viewer';
+import React from 'react';
+
+export enum ImageViewerMode {
+  inline = 'inline',
+  fullScreen = 'fullScreen',
+}
+
+export interface ImageViewerProps extends React.HTMLAttributes<HTMLDivElement> {
+  img: string;
+  hiResImg?: string;
+  width?: string;
+  height?: string;
+  zoom?: number;
+  defaultZoom: number;
+  maxZoom?: number;
+  snapView?: boolean;
+  refreshOnResize?: boolean;
+  zoomOnMouseWheel?: boolean;
+  hasZoomButtons?: boolean;
+  zoomStep?: number;
+  onLoad?: (viewer: any) => void;
+  onZoomChange?: (data: any) => void;
+}
+export interface FullScreenViewerProps
+  extends React.ImgHTMLAttributes<HTMLImageElement> {
+  img: string;
+  hiResImg?: string;
+  defaultZoom: number;
+  maxZoom?: number;
+  snapView?: boolean;
+  refreshOnResize?: boolean;
+  zoomOnMouseWheel?: boolean;
+  hasZoomButtons?: boolean;
+  zoomStep?: number;
+  onLoad?: (viewer: any) => void;
+}
+
+export interface ViewerState {
+  container: HTMLDivElement;
+  instance: BaseImageViewer;
```

Comment:
**Type bug**
ViewerState.instance would be different based on FullScreenViewer or ImageViewer.